### ISSUE TITLE
Repeated path segments

### ISSUE THREAD

**Issue by d-torrance** [Macaulay2/M2#1349]
A newly-introduced "repeated-path-segment" Lintian warning started popping up for the draft of the Debian package:

> The file is installed into a location that repeats the given
> path segment. An example would be `/usr/lib/lib` or
>  `/usr/share/myprogram/share`.
>
> More often than not this is unintended.

Here are the files in questions (with the repeated `EXA`):

```
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/EXA/Vision/EXA/luke-oeding.m2
M2/Macaulay2/packages/NumericalAlgebraicGeometry/SYSTEMS/EXA/Vision/EXA/luke-oeding-noSLPs.m2
```

Are these paths intentional?

**Comment by DanGrayson** [comment]
I bet those files aren't even used.  Let's do something about this.  @antonleykin ?

**Comment by d-torrance** [comment]
Closing, as this was fixed in #1353.
